Cumulative time limit clarification (#1114)
thewca/wca-regulations: wca-guidelines.md

- A1a2+++) [ADDITION] If the result from an attempt is unavailable (e.g. the timer was reset), the judge should consult the WCA Delegate and make an estimate of the elapsed time for the attempt. This estimate counts towards the time limit, but must not be used as the basis of an official result for the individual attempt.
- A1a2++++) [EXAMPLE] If a round has a cumulative time limit of 20 minutes and a time limit of 8 minutes, and the competitor's results of the first two attempts are 7:00 and 7:30, then the time limit for the third attempt is the lowest time of 20:00 - 7:00 - 7:30 = 5:30 and 8:00, i.e. 5:30.
- A1a2+++++) [CLARIFICATION] If a competitor reaches a cumulative time limit, the result for each of their remaining attempts in the round is recorded as DNS (Did Not Start).

> - A1a2++++++) [CLARIFICATION] If a round has a cumulative time limit applied to a combination of rounds for different events, the order of which the attempts are done is the order of the individual time limits counting towards the cumulative time limit.

Reviewer: ```suggestion
- A1a2++++++) [CLARIFICATION] If a cumulative time limit applies to a combination of rounds for different events, the order in which attempts are done is the order in which those attempts count towards the cumulative time limit.
```

Suggestion for clarity
